packages/smart-contracts/scripts-create2/contract-setup/setupBatchPayments.ts

```diff
@@ -23,24 +23,18 @@ export const setupBatchPayments = async (
   );
   await Promise.all(
     hre.config.xdeploy.networks.map(async (network) => {
```

Comment:
I am having a concern regarding the mapping, I used a loop on my branch to make logs easier to read:
logs:
network: mainnent
update ERC20 address into blabla
update Eth address into blabla
...
network: matic
update ERC20 address into blabla
update Eth address into blabla
...

With a mapping, is it still readable?